pkg/controller/inferenceservice/controller_test.go

```diff
@@ -372,7 +362,7 @@ func TestInferenceServiceWithOnlyPredictor(t *testing.T) {
 		if isvc.Status.GetCondition(apis.ConditionReady) == nil {
 			return true // Because ConditionReady might be removed, this is true
 		} else if isvc.Status.GetCondition(apis.ConditionReady).Status ==
-			v1.ConditionFalse {
+			v1.ConditionTrue {
```

Comment:
this should be still false as the test above is updating the condition to be false. I not sure why this sometimes fails the test and can not replicate locally.

Reply:
I run this several times, sometime this is false, and sometime is true. I found that, this condition is `True` in older version.